packages/trip-details/src/index.tsx

```diff
@@ -0,0 +1,232 @@
+import coreUtils from "@opentripplanner/core-utils";
+import moment from "moment";
+import React, { ReactElement } from "react";
+import { FormattedMessage, FormattedNumber } from "react-intl";
+import { CalendarAlt, Heartbeat, MoneyBillAlt } from "styled-icons/fa-solid";
+
+import * as Styled from "./styled";
+import TripDetail from "./trip-detail";
+import { CaloriesDetailsProps, TripDetailsProps } from "./types";
+
+// Load the default messages.
+import defaultMessages from "../i18n/en-US.yml";
```

Comment:
Since we're assuming the use of react-intl, I think we shouldn't bother using default messages.

Reply:
I think default messages are still useful if implementers don't provide their own messages (e.g. while getting started).